src/Features/Lsif/Generator/App.config

```diff
@@ -0,0 +1,5 @@
+﻿<?xml version="1.0" encoding="utf-8" ?>
+<configuration>
+  <!-- Use server GC for better performance: we care about throughput rather than responsiveness. -->
+  <gcServer enabled="true" />
```

Comment:
We're not, yet. This is (in some modes) running MSBuildWorkspace, which since that loads the build tasks in-proc many things don't work if we target .NET Core.

In my excitement, I started the project .NET Core, and then immediately ran into this and was sad.

Reply:
Interesting. AnalyzerRunner also uses MSBuildWorkspace, but is working on netcoreapp3.1.

Reply:
@sharwell Are you able to run AnalyzerRunner on Roslyn? Or are there any specific projects you know it _doesn't_ work for? It's quite possible my impression for what works these days is out of date.